git_machete/git_operations.py

```diff
@@ -43,16 +43,16 @@ def __init__(self, cli_opts: CommandLineOptions) -> None:
     def run_git(git_cmd: str, *args: str, **kwargs: Dict[str, str]) -> int:
         exit_code = utils.run_cmd("git", git_cmd, *args, **kwargs)
         if not kwargs.get("allow_non_zero") and exit_code != 0:
-            raise MacheteException(f"`{utils.cmd_shell_repr('git', git_cmd, *args, **kwargs)}` returned {exit_code}")
+            raise MacheteException(f"`{utils.get_cmd_shell_repr('git', git_cmd, *args, **kwargs)}` returned {exit_code}")
         return exit_code
 
     @staticmethod
-    def popen_git(git_cmd: str, *args: str, **kwargs: Dict[str, str]) -> str:
-        exit_code, stdout, stderr = utils.popen_cmd("git", git_cmd, *args, **kwargs)
+    def get_popen_git(git_cmd: str, *args: str, **kwargs: Dict[str, str]) -> str:
+        exit_code, stdout, stderr = utils.get_popen_cmd("git", git_cmd, *args, **kwargs)
         if not kwargs.get("allow_non_zero") and exit_code != 0:
-            exit_code_msg: str = fmt(f"`{utils.cmd_shell_repr('git', git_cmd, *args, **kwargs)}` returned {exit_code}\n")
-            stdout_msg: str = f"\n{utils.bold('stdout')}:\n{utils.dim(stdout)}" if stdout else ""
-            stderr_msg: str = f"\n{utils.bold('stderr')}:\n{utils.dim(stderr)}" if stderr else ""
+            exit_code_msg: str = get_fmt(f"`{utils.get_cmd_shell_repr('git', git_cmd, *args, **kwargs)}` returned {exit_code}\n")
```

Comment:
TBH I'd make an exception for `fmt`, `bold`, `dim`, `underline`, `colored`

Reply:
Since it's pretty clear that they do NOT have side effects, and this `get_` prefix makes formatted strings unnecessarily verbose